src/functionaltests/java/com/ericsson/ei/utils/DataBaseManager.java

```diff
@@ -145,12 +146,14 @@ public boolean verifyAggregatedObjectExistsInDB() throws InterruptedException {
     private List<String> compareSentEventsWithEventsInDB(List<String> checklist) {
         mongoClient = new MongoClient(getMongoDbHost(), getMongoDbPort());
         MongoDatabase db = mongoClient.getDatabase(database);
+        System.out.println("*******checklist*******"+checklist);
         MongoCollection<Document> collection = db.getCollection(eventMapCollection);
         List<Document> documents = collection.find().into(new ArrayList<>());
         for (Document document : documents) {
             for (String expectedID : new ArrayList<>(checklist)) {
-                if (expectedID.equals(document.get("_id").toString())) {
-                    checklist.remove(expectedID);
+            	Document objects = (Document) document.get("objects")) 
```

Comment:
Shouldn't this be "events"?

Reply:
Travis complains a lot about compilation errors, which can be discovered earlier if you run the maven commands in your local environment.

Reply:
It should be events. Just wanted to check why the travis has been failing.